pyrasterframes/src/main/python/README.md

```diff
@@ -1,39 +1,49 @@
 # PyRasterFrames
 
-PyRasterFrames is the Python API for Spark RasterFrames.
+PyRasterFrames is a library for distributed processing of geospatial raster data with Spark.
 
 
 ## Prerequisites
 
-1. [`sbt`](https://www.scala-sbt.org/)
+1. [`pip`](https://pip.pypa.io/en/stable/installing/)
 2. ['pyspark`](https://pypi.org/project/pyspark/) > 2.3.2 
 
-RasterFrames is primarily implmented in Scala, and as such uses the Scala build tool [`sbt`](https://www.scala-sbt.org/).
-All `sbt` commands referenced below must be run from the root source directory, i.e. the parent of the `pyrasterframes` 
-directory, including Python-related build steps. 
-
-As a tip, know that `sbt` is much faster if run in "interactive" mode, where you launch `sbt` with no arguments, 
-and subsequent commands are invoked via an interactive shell. But for context clarity, we'll prefix each command 
-example below with `sbt`. 
-
 ## Quickstart
 
-The quickest way to run a `pyspark` shell with the latest RasterFrames enabled is to run:
+The quickest way to get started is to `pip` install the pyrasterframes package.
 
 ```bash
-sbt pySparkCmd
+pip install pyrasterframes
+```
+
+You can then access a [`pyspark SparkSession`]() using the [`local[*]` master](https://spark.apache.org/docs/latest/submitting-applications.html#master-urls) as follows.
+
+```python
+import pyrasterframes
+spark = pyrasterframes.get_spark_session()
+```
+
+Then you can read a raster and do some simple processing on it.
+
+```python
+from pyrasterframes.rasterfunctions import *
+from pyspark.sql.functions import lit
+# Read a Landsat 8 L1TP PDS scene
+df = spark.read.rastersource('https://landsat-pds.s3.amazonaws.com/c1/L8/038/037/LC08_L1TP_038037_20190322_20190403_01_T1/LC08_L1TP_038037_20190322_20190403_01_T1_B4.TIF')
+# Add 3 to every cell, show some rows of the dataframe
+df.select(rf_local_add(df.tile, lit(3))).show(6, False)
```

Comment:
Such examples we'd like to make sure run fast even on modest hardware and slow network. File is ~60MB so small relative to lots of raster data, but we should try this in different settings to make sure it is responsive.